docs/config-files.md

```diff
@@ -7,11 +7,19 @@ id: config-files
 
 Babel has two parallel config file formats, which can be used together, or independently.
 
-* Project-wide configuration
-  * `babel.config.json` files, with the different extensions
-* File-relative configuration
-  * `.babelrc.json` files, with the different extensions
-  * `package.json` files with a `"babel"` key
+<details>
+  <summary>History</summary>
+| Version | Changes |
+| --- | --- |
+| `v7.8.0` | Support `.babelrc.mjs` and `babel.config.mjs` |
+| `v7.7.0` | Support `.babelrc.json`, `.babelrc.cjs`, `babel.config.json`, `babel.config.cjs` |
+</details>
```

Comment:
More a question than a comment to handle: would it be possible to use MDX, and generate this tables with something like
```js
<History versions={{
  "7.8.0": "Support `.babelrc.mjs` and `babel.config.mjs`",
  "7.7.0": "Support `.babelrc.json`, `.babelrc.cjs`, `babel.config.json`, `babel.config.cjs`"
}} />
```

Reply:
yeah wouldn't be hard, just a component if we move to v2. I wouldn't really worry about this though since it's not often we modify this anyway?

Reply:
Oh if it requires us to upgrade docusarus than it's definitely not a priority :joy: